app/helpers/view_components/site_header.rb

```diff
@@ -17,15 +17,16 @@ def to_s
     end
 
     def announcement_bar
-      return tag.span("") if !user_signed_in? ||
-                             current_user.donated? ||
-                             current_user.solutions.count < 3
+      return tag.span("") unless user_signed_in?
+      return tag.span("") if current_user.solutions.count <= 5
+      return tag.span("") if current_user.current_subscription
+      return tag.span("") if current_user.donated_in_last_35_days?
 
       link_to(Exercism::Routes.insiders_path, class: "announcement-bar md:block hidden") do
```

Comment:
does it correctly take you to `/insiders`?

Reply:
Yep, it will do.